api/src/main/java/com/velocitypowered/api/scheduler/Scheduler.java

```diff
@@ -23,29 +27,61 @@ public interface Scheduler {
    * @param runnable the task to run when scheduled
    * @return the task builder
    */
-  TaskBuilder buildTask(Object plugin, Runnable runnable);
+  TaskBuilder buildTask(@NotNull Object plugin, @NotNull Runnable runnable);
+
+  /**
+   * Initializes a new {@link TaskBuilder} for creating a task on the proxy.
+   *
+   * @param plugin the plugin to request the task for
+   * @return the task builder
+   */
+  TaskBuilder builder(@NotNull Object plugin);
```

Comment:
In the original issue, @astei talked about methods to create a builder that take all mandatory arguments are more idiomatic. Could we use the same approach and replace this by a `buildTask(Object, Consumer<ScheduledTask>)` method?

I rarely see the need for the `TaskBuilder#task` methods below, but I guess it makes the builders reusable (this should be documented).

Reply:
Now that you mention it, I would have liked the idea that the builder could be reusable, but I find its usage very rare and somewhat confusing